src/ir/auth_logic/lowering_ast_datalog.cc

```diff
@@ -133,17 +139,17 @@ LoweringToDatalogPass::BaseFactToDLIRInner(
 std::pair<datalog::Predicate, std::vector<datalog::Rule>>
 LoweringToDatalogPass::BaseFactToDLIRInner(const Principal& speaker,
                                            const Attribute& attribute) {
-  return std::make_pair(AttributeToDLIR(attribute),
-                        std::vector<datalog::Rule>(
-                            {SpokenAttributeToDLIR(speaker, attribute)}));
+  return std::make_pair(
+      AttributeToDLIR(attribute),
```

Comment:
nit: preferable to keep formatting changes in a separate PR.

Reply:
These format changes were unintentional!! Got auto-formatted on save.